### PR TITLE
Fix planned auctions not included in Your Auctions

### PR DESCRIPTION
**Pull request by Velenir**
Also check for sellVolumeNext when `indicesWithSellerBalance` is empty

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -626,12 +626,21 @@ export const getSellerOngoingAuctions = async (
       let ongoingAuction: AuctionObject
 
       const { sell: { decimals }, buy: { decimals: decimalsInverse } } = auction
-      if (indicesWithSellerBalance.length >= 1 || indicesWithSellerBalanceInverse.length >= 1) {
+      if (
+        indicesWithSellerBalance.length >= 1 || indicesWithSellerBalanceInverse.length >= 1 ||
```
Comment (W3stside) [inline src/api/index.ts]: just use `>=` 0 here

Comment (Velenir) [reply src/api/index.ts]: should there even be `if`

Comment (W3stside) [reply src/api/index.ts]: haha

File: src/api/index.ts [hunk 2]
```diff
@@ -625,56 +625,41 @@ export const getSellerOngoingAuctions = async (
...
-        const committedToNextNormal = sellVolumeNext.normal.gt(0)
-        const committedToNextInverse = sellVolumeNext.inverse.gt(0)
-
-        if (  // if there are truly no auctions with sellBalance
-          !committedToNextNormal && !committedToNextInverse &&
-          indicesWithSellerBalance.length === 0 && indicesWithSellerBalanceInverse.length === 0
-        ) return accum
-
-        let latestIndicesNormal: BigNumber[] = indicesWithSellerBalance,
+      const { sell: { decimals }, buy: { decimals: decimalsInverse } } = auction\
```
Comment (W3stside) [inline src/api/index.ts]: / here

**Review by W3stside — APPROVED** [review]

**Pull request merged**